.travis.yml

```diff
@@ -45,7 +45,6 @@ jobs:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
     - stage: "Additional Tests"
-      env: EMBER_TRY_SCENARIO=ember-lts-2.16
```

Comment:
+1 for ceasing to support this version of ember.

Had trouble catching catching errors thrown by the `render` function using `setupOnerror`.

Reply:
Same issue faced in #275

Reply:
I guess the solution to this in both PRs is either of:

1. Skip tests that override `Ember.onerror` in `2.16` test runs
2. Drop support for `2.16`

@Alonski @jelhan which would you prefer?